.travis.yml

```diff
@@ -10,8 +10,16 @@ node_js:
 - '5'
 - '4'
 - '0.12'
-script: make test-ci
+script: 
+  - 'if [ -n "${LINT-}" ]; then make lint ; fi'
+  - 'make test-ci'
```

Comment:
I wonder if we can skip `npm install` in that case (not that it would matter much)